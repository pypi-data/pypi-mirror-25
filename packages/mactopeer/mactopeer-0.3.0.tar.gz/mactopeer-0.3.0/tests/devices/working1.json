[
{
  "hostname": "192.168.0.1",
  "vendor": "ios",
  "optional_args": {
    "transport": "telnet",
    "port": 23
  }
}, {
  "hostname": "router1.example.com",
  "pmacct_ip": "10.0.0.1",
  "vendor": "eos",
  "use_peeringdb": true
}, {
  "hostname": "2001:db8::1",
  "pmacct_ip": "172.16.0.1",
  "vendor": "ios",
  "username": "test",
  "password": "test"
}
]
