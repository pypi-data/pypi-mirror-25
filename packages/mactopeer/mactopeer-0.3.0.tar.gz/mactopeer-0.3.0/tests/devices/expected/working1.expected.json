[
{
  "hostname": "192.168.0.1",
  "vendor": "ios",
  "optional_args": {
    "transport": "telnet",
    "port": 23
  }
},
{
  "hostname": "router1.example.com",
  "vendor": "eos",
  "pmacct_ip": "10.0.0.1",
  "use_peeringdb": true
},
{
  "hostname": "2001:db8::1",
  "username": "test",
  "password": "test",
  "vendor": "ios",
  "pmacct_ip": "172.16.0.1"
}
]
